src/utils.js

```diff
@@ -266,6 +266,27 @@ genotet.utils.swap = function(arr, i, j) {
 };
 
 /**
+ * Converts an array of string or object with string keys to a set
+ * with the strings as keys. The values will be set to all true.
+ * @param {!Array<string>|!Object<*>} collection
+ * @return {!Object<boolean>}
+ * @private
+ */
+genotet.utils.keySet_ = function(collection) {
```

Comment:
Make this public, or expose it via underscore then do _.keySet().

If latter, add the following to utils.js.
```js
/**
 * Initializes the utils. Creates underscore mixins.
 */
genotet.utils.init = function() {
  _.mixin({
    keySet: genotet.utils.keySet_,
  });
};
```